### PR TITLE
tests: drivers: uart: Enable test execution on nRF54L20pdk

### PR DESCRIPTION
**Pull request by nordic-segl**
Add overlays needed to execute uart driver tests on nRF54L20pdk.

### CODE REVIEW

**Review by nashif — APPROVED** [review]

**Comment by nordic-segl** [comment]
FYI: @e-rk

File: tests/drivers/uart/uart_elementary/testcase.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ tests:
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l20pdk/nrf54l20/cpuapp
       - nrf54l15dk/nrf54l15/cpuflpr
+      - nrf54l20pdk/nrf54l20/cpuapp
```
Comment (e-rk) [inline tests/drivers/uart/uart_elementary/testcase.yaml:17]: `nrf54l20` is already there in the list just two lines above.

File: tests/drivers/uart/uart_pm/testcase.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ common:
     - nrf54l15dk/nrf54l15/cpuapp
     - nrf54l20pdk/nrf54l20/cpuapp
     - nrf54l15bsim/nrf54l15/cpuapp
+    - nrf54l20pdk/nrf54l20/cpuapp
```
Comment (e-rk) [inline tests/drivers/uart/uart_pm/testcase.yaml:11]: Same here

File: tests/drivers/uart/uart_pm/testcase.yaml [hunk 3]
```diff
@@ -35,6 +36,7 @@ tests:
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l20pdk/nrf54l20/cpuapp
       - nrf54l15bsim/nrf54l15/cpuapp
+      - nrf54l20pdk/nrf54l20/cpuapp
```
Comment (e-rk) [inline tests/drivers/uart/uart_pm/testcase.yaml:39]: Same here

**Review by e-rk — CHANGES_REQUESTED** [review]

**Comment by nordic-segl** [comment]
Changes are already applied (other PR was merged in mean time).

**Pull request closed without merging**